cmd/gateway-startup-msg.go

```diff
@@ -39,6 +39,9 @@ func printGatewayStartupMessage(apiEndPoints []string, backendType string) {
 	// first endpoint as default.
 	printCLIAccessMsg(strippedAPIEndpoints[0], fmt.Sprintf("my%s", backendType))
 
+	// Print TLS fingerprints for self-singed certificates
+	printTLSFingerprints()
```

Comment:
We should talk about in our documentation how these fingerprints are useful for the client

Reply:
Have added an additional block to the TLS doc. However, the fingerprints become really useful when `mc` supports fingerprint pinning. Then you can do `mc ...` on a `minio ... --auto-tls ...`without `--insecure` or anything. `mc` will simply work regardless whether it's HTTP, HTTPS ("self-signed") or HTTPS